src/toil/lib/docker.py

```diff
@@ -0,0 +1,255 @@
+import base64
+import logging
+import subprocess
+
+import os
+from bd2k.util.exceptions import require
+
+_logger = logging.getLogger(__name__)
+
+
+def dockerCall(job,
+               tool,
+               parameters=None,
+               workDir=None,
+               dockerParameters=None,
+               outfile=None,
+               defer=None):
+    """
+    Wrapper call to Docker. See docstring for _docker()
```

Comment:
Would you prefer I copy and paste the `_docker` docstring into both `dockerCall` and `dockerCheckOutput` and then remove all but the `:param` stuff from `_docker` ?